projects/js-packages/publicize-components/src/hooks/use-anaytics-tracks/README.md

```diff
@@ -0,0 +1,24 @@
+# useAnalyticsTracks hook
+This hook is used to add some functionality around tracks analytics
+Handles initializing the user and allows for easy setup of callbacks
+This implementation only records events when the current user has a Jetpack connection
```

Comment:
From what I understand, the main feature of this new hook is just to limit recording events to the situation "[...] when the current user has a Jetpack connection" and it combines the page view event triggered on the first render, with an alias for recording further events.

I'm not sure if that's necessary, especially since you just implemented it in only one component. However, if you believe it's worth having it - I would describe and/or name it more explicitly.